fix_sets/set_cats.py

```
@@ -14,23 +14,24 @@
 import tqdm
-from api_bots import printe
+
 from fix_mass.files import studies_titles, studies_titles2
 from fix_sets.bots2.filter_ids import filter_no_title
 from fix_sets.bots2.text_cat_bot import add_cat_to_set
 from fix_sets.ncc_api import CatDepth, ncc_MainPage
-
+import logging
+logger = logging.getLogger(__name__)
 
 def work_one_study(study_id, study_title="", categories=[]):
```

Ruff (0.15.0)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)